core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1180,6 +1180,14 @@ package object config {
       .intConf
       .createWithDefault(1)
 
+  private[spark] val IO_WARNING_LARGEFILETHRESHOLD =
+    ConfigBuilder("spark.io.warning.largeFileThreshold")
+      .internal()
+      .doc("When spark loading one single large file, if file size exceed this " +
```

Comment:
If the size in bytes of a file loaded by Spark exceeds this threshold, a warning is logged with the possible reasons.